_sass/_layout.scss

```diff
@@ -498,69 +517,58 @@
 
 .post-header {
     margin-bottom: $spacing-unit;
+    padding: 30px 0;
+    background: #f5f5f5;
```

Comment:
given the fact that we want to adopt to mozilla style colors in the future, could you please add this as a variable?